### PR TITLE
Blog app - views

### PR DESCRIPTION
**Pull request by khanh-devos**
- [x] Implement the design from the sneak peek wireframes.
- [x] Styling is nice to have but focus on making sure everything functions first. (Note: Ignore login logic, you would be using Devise in coming milestone)

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <p style="color: green"><%= notice %></p>
```
Comment (Lucas-Masaba) [inline app/views/users/show.html.erb:1]: - [x] You should use the `#recent_post` method you created in your model to render the user's 3 most recent posts.

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -7,17 +7,17 @@
   <%= render @user %>
 
   <% @posts.each do |post| %>
-    <%= link_to show_a_user_a_post_comments_path(id: post.id) do %>
+    <%= link_to "/users/#{@user.id}/posts/#{post.id}" do %>
```
Comment (Lucas-Masaba) [inline app/views/posts/index.html.erb:10]: - [x] Use the `recent_comments` method you created in your Post model to render the 5 most recent comments here.

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @khanh-devos  👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Highlights 🍾 
- Linter checks passing :heavy_check_mark: 
- Views are rendering on the browser :gear: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### [OPTIONAL]
- You have a bunch of commented code on your controllers and views that I cannot directly comment on in this review. I would suggest removing them if you are not using them.
- I also noticed that you do not have the photo field in your schema which should be there. I can't add photos when creating a user and so they do not appear in the UI.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by khanh-devos** [comment]
Thank you for your quick review, @Lucas-Masaba

**Review by gemmen29 — APPROVED** [review]
Hi @khanh-devos, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**